doc/content/getting-started/upgrading.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Upgrading"
+description: ""
+weight: -1
```

Comment:
Isn't this putting the upgrading section before anything else?

Reply:
Installation also has weight -1, so it's alphabetical between those and then everything else after